alerts/charts/templates/am-base-config.yaml

```diff
@@ -58,4 +58,23 @@ spec:
             name: {{ $.Release.Name }}-webhook-urls
```

Comment:
how was this secret provided earlier, before it has been attached to the config here?

Reply:
It was remove accidentally.